roles/precheck/tasks/main.yml

```diff
@@ -14,3 +14,11 @@
     _hosts: >-
       {{ (groups.get(node_group | d('node'), []) + groups.get(frontend_group | d('frontend'), [])) | unique }}
   when: inventory_hostname in _hosts
+
+- ansible.builtin.include_tasks:
+    file: "{{ role_path }}/tasks/pci_passthrough.yml"
+  vars:
+    _hosts: "{{ groups.get(node_group | d('node'), []) | unique }}"
```

Comment:
See here you use d() and in other places default(), I think using d() everywhere is ok practice.